.circleci/config.yml

```diff
@@ -9,10 +9,14 @@ jobs:
       # CircleCI maintains a library of pre-built images
       # documented at https://circleci.com/docs/2.0/circleci-images/
       - image: circleci/postgres:9.4
+        environment:
```

Comment:
@coderdan the example configs in circle seem to hint that env vars need to be set per image, too. I noticed master was failing since the last few pulls but not sure what could have changed to make this an issue.